### ISSUE TITLE
is_training key with rllib

### ISSUE THREAD

**Issue by cometta** [ray-project/ray#9569]
ray 0.8.6, python 3.7, pytorch 1.5.1, using rllib PPO algorithm 

During, training, the output of input_dict["is_training"] in ```  def forward(self, input_dict,  ... ``` is starting returning False,False, False then becomes True, True True for each iteration. Aren't it suppose to be True all the time?

**Comment by sven1977** [comment]
It should be False when the Model call happens from out of the compute_actions method of the Policy (sampling/inference) and True when the Model is called in some loss function using a train_batch.
